monkey/infection_monkey/island_api_client/http_island_api_client.py

```diff
@@ -66,6 +66,14 @@ def get_agent_plugin(self, plugin_type: AgentPluginType, plugin_name: str) -> Ag
 
         return AgentPlugin(**response.json())
 
+    @handle_response_parsing_errors
+    def get_agent_plugin_manifest(
+        self, plugin_type: AgentPluginType, plugin_name: str
+    ) -> AgentPluginManifest:
+        response = self.http_client.get(f"agent-plugins-manifest/{plugin_type.value}/{plugin_name}")
```

Comment:
Did you test this?

https://github.com/guardicore/monkey/blob/d5585865731ffd912047477f8591487e7cd2c4aa/monkey/monkey_island/cc/resources/agent_plugins_manifest.py#L13-L17

Reply:
I have unit tests which passed 😕